test/Microsoft.AspNetCore.Authentication.Test/GoogleTests.cs

```diff
@@ -1061,19 +1061,19 @@ private static TestServer CreateServer(Action<GoogleOptions> configureOptions, F
                 .ConfigureServices(services =>
                 {
                     services.AddTransient<IClaimsTransformation, ClaimsTransformer>();
-                    services.AddAuthentication(o =>
-                    {
-                        o.DefaultScheme = TestExtensions.CookieAuthenticationScheme;
-                        o.DefaultChallengeScheme = GoogleDefaults.AuthenticationScheme;
-                    });
-                    services.AddAuthentication()
+                    services.AddAuthentication("Auth")
+                        .AddVirtualScheme("Auth", "Auth", o =>
+                        {
+                            o.DefaultTarget = TestExtensions.CookieAuthenticationScheme;
+                            o.ChallengeTarget = GoogleDefaults.AuthenticationScheme;
+                        })
                         .AddCookie(TestExtensions.CookieAuthenticationScheme)
                         .AddGoogle(configureOptions)
                         .AddFacebook(o =>
```

Comment:
Why do the google tests include facebook?

Reply:
There's a test that ensures that when you ask for google but get a facebook identity it throws I think,